concent_api/utils/helpers.py

```diff
@@ -113,18 +113,22 @@ def sign_message(golem_message, priv_key):
 
 
 def get_storage_file_path(category, subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None and category is not None
     return f'blender/{category}/{task_id}/{task_id}.{subtask_id}.zip'
 
 
 def get_storage_result_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
     return get_storage_file_path('result', subtask_id, task_id)
 
 
 def get_storage_scene_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
     return get_storage_file_path('scene', subtask_id, task_id)
 
 
 def get_storage_source_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
```

Comment:
those asserts are not needed, it is checked in `get_storage_file_path`

Reply:
Done